hr_timesheet_name_customer/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2015-17 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
+
+{
+    "name": "Name Customer",
+    "summary": "Add ‘Description Customer’ field for timesheets",
+    "version": "16.0.1.0.0",
+    "category": "Timesheet",
+    "website": "https://github.com/OCA/timesheet",
+    "author": "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["hr_timesheet"],
+    "data": [
```

Comment:
Add "maintainers" key with:
- CetmixGitDrone
- You

See example [here](https://github.com/cetmix/oca-account-payment/blob/bbede6c4d155b7ff0809abbae717e0f6bc678780/product_restrict_payment_acquirer/__manifest__.py#L8)